tachyon/crypto/commitments/merkle_tree/field_merkle_tree/field_merkle_tree_mmcs.h

```diff
@@ -23,6 +28,8 @@ class FieldMerkleTreeMMCS final
  public:
   using PrimeField = typename PackedPrimeField::PrimeField;
   using Commitment = std::array<PrimeField, N>;
+  using Digest = Commitment;
```

Comment:
Is there a reason for renaming Commitment as Digest? It seems like they basically mean the same thing.

Reply:
This is because it looks strange to me that `Proof` is a collection of the `Commitment`, but to remove `Digest` is also fine with me. I'll remove it.

Reply:
Let's just keep it this way. Maybe this is clearer.